tensorboard/uploader/uploader_test.py

```diff
@@ -50,6 +51,19 @@
 from tensorboard.util import test_util as tb_test_util
 
 
+def _create_example_graph(test_attr_size):
```

Comment:
The arg name `test_attr_size` is potentially misleading. How about calling it `large_test_attr_size` to reflect the fact that it controls only one of the two attrs that is meant to be bigger?

Reply:
Done as `large_attr_size`